chronos/internal/asyncfutures.nim

```diff
@@ -60,7 +60,7 @@ type
   FutureCompletedError* = object of FutureError
     ## Error raised when trying access the error of a completed Future
 
-  SomeFuture = Future|InternalRaisesFuture
+  SomeFuture* = Future|InternalRaisesFuture
```

Comment:
this type is deliberately private because the whole idea of how this is implemented might change soonish - for now, it's better to use `InternalRaisesFuture` directly in code that needs it which will make it easy to find, once a stable future type is created.